activitysim/examples/example_mtc_extended/test/test_mtc_extended.py

```diff
@@ -0,0 +1,57 @@
+# ActivitySim
+# See full license in LICENSE.txt.
+import os
+import subprocess
+import pkg_resources
+
+import pandas as pd
+import pandas.testing as pdt
+
+from activitysim.core import inject
+
+
+def teardown_function(func):
+    inject.clear_cache()
+    inject.reinject_decorated_tables()
+
+
+def test_mtc_extended():
+
+    def example_path(dirname):
+        resource = os.path.join('examples', 'example_mtc_extended', dirname)
+        return pkg_resources.resource_filename('activitysim', resource)
+
+    def example_mtc_path(dirname):
+        resource = os.path.join('examples', 'example_mtc', dirname)
+        return pkg_resources.resource_filename('activitysim', resource)
+
+    def test_path(dirname):
+        return os.path.join(os.path.dirname(__file__), dirname)
+
+    def regress():
+        regress_trips_df = pd.read_csv(test_path('regress/final_trips.csv'))
+        final_trips_df = pd.read_csv(test_path('output/final_trips.csv'))
+
+        regress_vehicles_df = pd.read_csv(test_path('regress/final_vehicles.csv'))
+        final_vehicles_df = pd.read_csv(test_path('output/final_vehicles.csv'))
+
+        # person_id,household_id,tour_id,primary_purpose,trip_num,outbound,trip_count,purpose,
+        # destination,origin,destination_logsum,depart,trip_mode,mode_choice_logsum
+        # compare_cols = []
+        pdt.assert_frame_equal(final_trips_df, regress_trips_df)
```

Comment:
This is not passing in my test run. The order of columns in `final_trips_df` seems to be different from `regress_trips_df`, other than that the two are identical. Perhaps `sort_index` before comparing.

Reply:
Sorry about that. I think the ordering got changed when I merged in the changes from develop?  Regardless, this is fixed.  The extended mtc example was added to the travis.yml so the test is run now as part of our CI testing.